bin/bkg_fixed_counts

```diff
@@ -173,6 +173,29 @@ def find_outer_bkg_radius(coords, bkg_x, bkg_y, numcts, maxradius):
         coords['outer_rad_pix'].append(outer_rad)
 
 
+def _remove_inner_circle(background, inner_circle):
+    """The ecf circle with the radius equal to the annulus'
+    inner radius is always excluded.  We don't want that.
+    """
+    from region import CXCRegion, opAND, opOR, opNOOP
+    idx = background.index(-inner_circle)
```

Comment:
Sp, by construction we know that inner_circle is going to be an exclude region in the background, which is why we don't need to check for the `index` method raising a `ValueError`?